sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClientBuilder.java

```diff
@@ -110,11 +116,13 @@ public CertificateClient buildClient() {
      * {@link CertificateClientBuilder#vaultUrl(String)} have not been set.
      */
     public CertificateAsyncClient buildAsyncClient() {
-        Configuration buildConfiguration = (configuration == null) ? Configuration.getGlobalConfiguration().clone() : configuration;
+        Configuration buildConfiguration = (configuration == null) ? Configuration.getGlobalConfiguration().clone()
+            : configuration;
```

Comment:
Let's use this pattern for readability

```java
variable = conditional
  ? passed_conditional
  : failed_conditional
```

Reply:
```java
Configuration buildConfiguration = (configuration != null) ? configuration: 
            Configuration.getGlobalConfiguration().clone();
```

Reply:
Made the change above.

Reply:
That won't pass linting, it expects `?` and `:` to either begin the line or be in the line, it expects the following:

```java
variable = conditional ? passed : failed;

variable = conditional ? passed
  : failed;

variable = conditional
  ? passed
  : failed;
```